src/tools/wasm-emscripten-finalize.cpp

```diff
@@ -226,6 +226,13 @@ int main(int argc, const char *argv[]) {
     WasmPrinter::printModule(&wasm, std::cerr);
   }
 
+  // Stip target features section (needed for metadata)
```

Comment:
typo in `strip`

also the part in parens is confusing - maybe "no longer needed after we get metadata from it"?